### PR TITLE
fix minor error in smoothstep function's documentation

### LINKED ISSUE #45804: C# Mathf.Smoothstep returns infinitesimal result
**Godot version:**

fe38afca89c453fe01591fedc0dc81a459a6927ea39b16d0d60cc2b2f583d0d74cfdbe6b83a9a4394250a237da3dc4079b31f828033fd1bec77c4547b9021ece  Godot_v3.2.4-rc1_win64.exe.zip

**OS/device including version:**

Windows 10 Pro 10.0.19041, GTX1080TI, Ryzen 2700x, GLES3, C#

**Issue description:**

The Mathf.SmoothStep function gives unexpected result. I'm expecting similar results to Mathf.Lerp(), but I am getting very small numbers back instead. The function seems to be  returning a number that correlates to its intended value, but it's scaled very small.

**Steps to reproduce:**
Use Mathf.SmoothStep to interpolate between a minimum and Maximum number.

**Minimal reproduction project:**

I've make a visual representation of the problem in this project.
[Smoothstep.zip](https://github.com/godotengine/godot/files/5939862/Smoothstep.zip)
image

### PR DESCRIPTION
**Pull request by angad-k**
fixes #45804
fixes a minor issue in the documentation for the smoothstep function which incorrectly mentioned ```x``` being directly fed into ```3*x^2 - 2*x^3``` whereas it was a scaled version of it according to the ```from``` and ```to``` arguments

### CODE REVIEW

File: doc/classes/@GlobalScope.xml [hunk 1]
```diff
@@ -935,7 +935,7 @@
 			<description>
 				Returns the result of smoothly interpolating the value of [code]x[/code] between [code]0[/code] and [code]1[/code], based on the where [code]x[/code] lies with respect to the edges [code]from[/code] and [code]to[/code].
 				The return value is [code]0[/code] if [code]x &lt;= from[/code], and [code]1[/code] if [code]x &gt;= to[/code]. If [code]x[/code] lies between [code]from[/code] and [code]to[/code], the returned value follows an S-shaped curve that maps [code]x[/code] between [code]0[/code] and [code]1[/code].
-				This S-shaped curve is the cubic Hermite interpolator, given by [code]f(x) = 3*x^2 - 2*x^3[/code].
+				This S-shaped curve is the cubic Hermite interpolator, given by [code]f(y) = 3*y^2 - 2*y^3[/code] where [code]y = (x-from)(to-from)[/code].
```
Comment (lawnjelly) [inline doc/classes/@GlobalScope.xml:938]: I suspect it should be `(x-from) / (to-from)`, you appear to have missed out the divide.

Comment (angad-k) [reply doc/classes/@GlobalScope.xml:938]: yes. missed it. Will update quickly

Comment (angad-k) [reply doc/classes/@GlobalScope.xml]: addressed this in a new PR

**Comment by Calinou** [comment]
Superseded by https://github.com/godotengine/godot/pull/45934.

**Pull request closed without merging**